codacy/values.yaml

```diff
@@ -297,11 +297,19 @@ crow:
 rabbitmq-ha:
   rabbitmqErlangCookie: "PLEASE_CHANGE_ME_TO_A_RANDOM_STR" # Generate one with `cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1`
   fullnameOverride: codacy-rabbitmq-ha
-  rabbitmqUsername: "rabbitmq"
-  rabbitmqPassword: "rabbitmq"
+  auth:
+    username: "rabbitmq-codacy"
+    password: "rabbitmq-codacy"
   prometheus:
     operator:
       enabled: false
+  podManagementPolicy: Parallel
+  forceBoot: true # Allows a graceful recovery from 'Waiting for Mnesia tables' in case all rabbitmq pods restart.
+  loadDefinition:
+    enabled: true
+    existingSecret: rabbitmq-load-definition
+  extraConfiguration: |
+    management.load_definitions = /app/load_definition.json
```

Comment:
Load the secret json file mentioned above to inject the queues that must always be created.